FRC-Extension/RoboRIOCode/DeployManager.cs

```diff
@@ -250,19 +230,19 @@ public CodeReturnStruct(string exe, List<string> files)
         public async Task<bool> DeployRobotFilesAsync(List<string> files)
         {
             await OutputWriter.Instance.WriteLineAsync("Deploying robot files").ConfigureAwait(false);
-            return await RoboRIOConnection.DeployFiles(files, DeployProperties.DeployDir, ConnectionUser.LvUser).ConfigureAwait(false);
+            return await m_roboRioConnection.DeployFiles(files, DeployProperties.DeployDir, ConnectionUser.LvUser).ConfigureAwait(false);
```

Comment:
You're blindly assuming that `m_roboRioConnection` isn't null. I'd suggest making that an assertion, to ensure it actually causes a noticeable error if it ever happens.